packages/next/src/build/webpack/plugins/middleware-plugin.ts

```diff
@@ -42,10 +42,10 @@ export interface EdgeFunctionDefinition {
   name: string
   page: string
   matchers: MiddlewareMatcher[]
+  environments: Record<string, string>
```

Comment:
That's a pretty weird name.

```suggestion
  env: Record<string, string>
```